server/src/DBTablesUser.cc

```diff
@@ -128,8 +128,8 @@ static const ColumnDef COLUMN_DEF_ACCESS_LIST[] = {
 	NULL,                              // defaultValue
 }, {
 	"host_group_id",                   // columnName
-	SQL_COLUMN_TYPE_BIGUINT,           // type
-	20,                                // columnLength
+	SQL_COLUMN_TYPE_VARCHAR,           // type
```

Comment:
In fact, I'm still thinking.
I have columns of which I'd like to change the type other than here, such as Trigegr ID, and Event ID.
Of course, providing automatic upgrade feature is better. However, it takes a time.
I don't known what is important for us. Adding features rapidly, Compatibility, Easy to use ?
If we add an automatic migration feature, we can do it after. So shall we ignore that in this pull request.